src/components/ButtonTag/ButtonTag.tsx

```diff
@@ -4,15 +4,35 @@ import style from "./ButtonTag.module.scss";
 type Props = {
     label?: string;
     icon: ReactNode;
+    onClick: () => void;
```

Comment:
Does onClick get overriden by `DetailedHTMLProps<
    ButtonHTMLAttributes<HTMLButtonElement>,
    HTMLButtonElement>`? Changing the parameter and return type i mean

Reply:
to avoid it is undefined, I have added the Omit